be/src/exec/tablet_sink.cpp

```diff
@@ -515,14 +513,31 @@ void NodeChannel::try_send_batch(RuntimeState* state) {
         CHECK(_pending_batches_num == 0) << _pending_batches_num;
     }
 
-    if (_parent->_transfer_data_by_brpc_attachment && request.has_row_batch()) {
-        request_row_batch_transfer_attachment<PTabletWriterAddBatchRequest,
-                                              ReusableClosure<PTabletWriterAddBatchResult>>(
-                &request, _tuple_data_buffer, _add_batch_closure);
+    if (config::transfer_large_data_by_brpc && request.has_row_batch() &&
```

Comment:
User can change this config at runtime. So better to save its value when creating tablet sink.
Otherwise, it may changed during loading process.

Reply:
fix, I understand the purpose of your previous code : )